monitoring-dashboard/components/org.wso2.ei.dashboard.bootstrap/src/main/java/org/wso2/ei/dashboard/bootstrap/DashboardServer.java

```diff
@@ -148,24 +145,58 @@ public void startServerWithConfigs() {
             System.exit(1);
         } catch (ConfigParserException e) {
             logger.error("Error while reading TOML file configs", e);
+            System.exit(1);
```

Comment:
This is correct, but a little bit dangerous to ship this in a patch. But let's keep it.